### PR TITLE
Added activate_block_id support for workbench

### PR DESCRIPTION
**Pull request by e-kolpakov**
This PR adds support for `activate_block_id` feature: [upstream PR](https://github.com/edx/edx-platform/pull/8768).

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @e-kolpakov! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request. 

To automatically create an OSPR issue for this pull request, just visit this link: http://openedx-webhooks.herokuapp.com/github/process_pr?number=64&repo=edx%2Fxblock-sdk

**Comment by edx-webhook** [comment]
Thanks for the pull request, @e-kolpakov! I've created [OSPR-731](https://openedx.atlassian.net/browse/OSPR-731) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've added yourself to the [AUTHORS](https://github.com/edx/xblock-sdk/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

**Comment by sarina** [comment]
@e-kolpakov do you need help debugging the failing tests?

**Comment by e-kolpakov** [comment]
@sarina I think I do. I'e just rerun Ci build on master and it fails, despite that when it was ran last time (2 months ago) it passed.

**Comment by sarina** [comment]
OK. I've asked Ned to take a look at the PR (he's release master this week so maybe he won't have time for a day or two). I'm sure he'll pull in whomever if he can't help you figure out the test failures.

**Comment by e-kolpakov** [comment]
@sarina thank you. It's not urgent - the changes only needed for test purposes, and it can wait for some time. As a last resort I can switch to use the version from the branch.

File: workbench/views.py [hunk 1]
```diff
@@ -64,7 +64,11 @@ def show_scenario(request, scenario_id, view_name='student_view', template='work
     usage_id = scenario.usage_id
     runtime = WorkbenchRuntime(student_id)
     block = runtime.get_block(usage_id)
-    frag = block.render(view_name)
+    render_context = {}
+    if request.GET.get('activate_block_id', None):
```
Comment (Kelketek) [inline workbench/views.py]: Would it hurt anything to always have this present and just do:
```
render_context['activate_block_id'] = request.GET.get('activate_block_id', None)
```

?

File: AUTHORS [hunk 2]
```diff
@@ -24,3 +24,4 @@ David Baumgold <david@davidbaumgold.com>
 Piotr Mitros <pmitros@edx.org>
 Steven Burch <stv@stanford.edu>
 Brad Smith <usernamenumber@gmail.com>
+Eugeny Kolpakov <eugeny.kolpakov@gmail.com>
```
Comment (Kelketek) [inline AUTHORS]: Nit: Newline

**Comment by e-kolpakov** [comment]
@sarina I've fixed the remaining test failures, so build is green and I'm about getting @Kelketek +1 on this, so this should be ready for reviewing very soon.

**Comment by sarina** [comment]
OK - I've already handed the ticket to the platform team, so review is in their court.

**Comment by Kelketek** [comment]
:+1:

**Comment by doctoryes** [comment]
@e-kolpakov I've tried to install the requirements for the solutions XBlock SDK and have discovered that the commit hash for the solutions' XBlock repo in the requirements.txt file (f0112919e625ef037d1f0759ceb6ef7353cf367d) doesn't seem to exist. It's not entirely relevant to this PR - but FYI...

**Comment by doctoryes** [comment]
@e-kolpakov Nevermind! I was on master, not your branch.

**Comment by doctoryes** [comment]
:+1:  LGTM. Going to merge next.

**Pull request merged**